Components/Entities/FeedbackLink.cs

```diff
@@ -21,5 +21,29 @@ public class FeedbackLink
         /// If it is empty will asume FeebackLinkurl is bilingual or also non-existant
         /// </summary>
         public string UrlFr { get; set; }
+
+        /// <summary>
+        /// Set a display name of your contact link within the feedback tool
+        /// Set by application via web.config 'FeedbackLinkText' or programmatically
+        /// </summary>
+        public string Text { get; set; }
+
+        /// <summary>
+        /// URL to be used for the contact link within the feedback tool
+        /// Set by application via web.config 'FeedbackLinkHref' or programmatically
+        /// </summary>
+        public string Href { get; set; }
```

Comment:
As discussed, we don't need Href because we have URL (and test should get a french?) (applies to config/settings as well)

Reply:
done